bundles/org.openhab.core.model.yaml/src/main/java/org/openhab/core/model/yaml/internal/YamlModelRepositoryImpl.java

```diff
@@ -208,27 +206,19 @@ public synchronized void processWatchEvent(Kind kind, Path fullPath) {
                     newElementNames.add(elementName);
                     JsonNode node = element.getValue();
 
-                    List<JsonNode> newNodeV1Elements = new ArrayList<>();
-                    JsonNode newNodeElements = null;
-
-                    if (modelVersion == 1) {
-                        if (!node.isArray()) {
-                            // all processable sub-elements are arrays
-                            logger.trace("Element {} in model {} is not an array, ignoring it", elementName, modelName);
-                            continue;
-                        }
-                        node.elements().forEachRemaining(newNodeV1Elements::add);
-                    } else {
-                        if (!node.isContainerNode() || node.isArray()) {
-                            // all processable sub-elements are container nodes (not array)
-                            logger.trace("Element {} in model {} is not a container object, ignoring it", elementName,
+                    if (!node.isContainerNode() || node.isArray()) {
+                        // all processable sub-elements are container nodes (not array)
+                        logger.trace("Element {} in model {} is not a container object, ignoring it", elementName,
```

Comment:
@lolodomo I think this should be a warn as well, I just tested my "old" V1 rules, and nothing was logged until I turned on trace. I know that the file is no longer valid, but using an array instead of a map is an "easy mistake to make" for users, and as it is logged it seems like everything is OK, but the element is ignored and doesn't show up.

Reply:
Probably not wished generally but could be a good idea in case of a node we are expecting.

Reply:
I don't know what "the general wishes" regarding logging are, but I think it's important that users get feedback when they make a mistake so that they can correct it. All that is logged is "Adding YAML model <file name>" (INFO) and then nothing that indicates that something went wrong. But the object never shows up in the system.

Reply:
I am not against a general warning finally.